pages/index.tsx

```diff
@@ -115,13 +115,6 @@ function index({
     setCountryCode({ setcountry, setcurrency, country });
   }, []);
 
-  // If project details are present set project details
-  React.useEffect(() => {
```

Comment:
This seems to be required to set the project details

Reply:
yes

Reply:
done